docs/streams/stream-basics/create-producers.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 30
+title: Create Producers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Prerequisites from '../../_partials/_prerequisites-sdk-api-key.md';
+
+This page describes how to create a producer.
+
+<Prerequisites />
+
+## Create Producer Code
+
+1. Change `BASE_URL` if necessary and insert `apiKey`.
```

Comment:
@jamesm-macrometa, please make this and the consumers instructions consistent with each other.